target/hexagon/meson.build

```diff
@@ -177,6 +177,7 @@ idef_parser = executable('idef-parser',
                           idef_parser_dir / 'parser-helpers.c'],
                          include_directories: ['idef-parser', '../../include/'],
                          dependencies: [glib_dep],
+						 c_args: ['-std=c11', '-Wextra', '-Wall'],
```

Comment:
Mind the indentation.
What's the current set of arguments?

You should be able to see it by:
```
ninja --verbose
```

Reply:
I'll remove the `C11` flag as I mentioned above to comply with coding standards, which means changing some macros around in `parser-helpers.h`. I'll remove '-Wall' as well since it is already included. However ´-Wextra' is not included by QEMU's build script, but I think it's good to keep for our code to get some better warnings.